### PR TITLE
DM-2848: Bypass ntlm auth call for public IPs

### PR DESCRIPTION
**Pull request by emyl3**
### JIRA issue link
[DM-2848](https://agile6.atlassian.net/browse/DM-2848)

## Description - what does this code do?
Bypass ntlm auth call for public IPs

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating JIRA issue
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by JoshingYou1 — APPROVED** [review]

File: proxy/Dockerfile [hunk 1]
```diff
@@ -55,4 +55,4 @@ EXPOSE 8080
 USER nginx
 STOPSIGNAL SIGTERM
 RUN nginx -V
-CMD ["nginx", "-e", "/home/nginx/log/error.log", "-g","daemon off;"]
\ No newline at end of file
+CMD ["nginx", "-e", "/home/nginx/log/error.log", "-g","daemon off", "--with-http_realip_module"]
```
Comment (ahay-agile6) [inline proxy/Dockerfile:58]: very interesting.
what does this module do?

Comment (emyl3) [reply proxy/Dockerfile:58]: I saw it as a way to get the real IP from the many stackoverflow questions we were searching through but I am going to remove it on dev to see if we actually need it 🤔 

https://nginx.org/en/docs/http/ngx_http_realip_module.html
https://serverfault.com/questions/314574/nginx-real-ip-header-and-x-forwarded-for-seems-wrong

Comment (emyl3) [reply proxy/Dockerfile:58]: that and also the stuff we added in the ngnix.conf

Comment (emyl3) [reply proxy/Dockerfile]: Good catch, Aurora! It was not needed so it has been removed :D

**Review by JoshingYou1 — APPROVED** [review]
Well, since we didn't need all of that header code, it's even cleaner now! Thanks Elisa and thank you @ahay-agile6 for catching that! 😄

**Pull request merged**